sqlmesh/integrations/github/cicd/controller.py

```diff
@@ -115,6 +115,35 @@ def is_skipped(self) -> bool:
         return self == GithubCheckConclusion.SKIPPED
 
 
+class BotCommand(Enum):
+    INVALID = 1
+    DEPLOY_PROD = 2
+
+    @classmethod
+    def from_comment_body(cls, body: str, namespace: t.Optional[str] = None) -> BotCommand:
+        body = body.strip()
+        command_aliases = {
+            cls.DEPLOY_PROD: {"/deploy_prod", "/deploy-prod", "/deployprod", "/deploy"},
```

Comment:
Do we need all these aliases? I remember multiple aliases is something we decided against for engine operators in airflow.

Reply:
This is why I think this is different: Config is something the user generally sets once and that is it. Therefore asking the user to check for specific syntax is fine since it is a low cost operation. Also if they do it incorrectly they will quickly see an error. 

In this case the user is providing an input potentially many times. It is in an interface we don't control and can't provide hints in. Also it takes some time to even know if the input was processed (the actions have to kick off and parse). So in this context I think it is good to be forgiving on syntax.

Reply:
what do you mean we don't control? there's no feedback for unregistered commands? 

fwiw i don't think we need all these aliases, probably just /deploy is good enough

Reply:
I removed the aliases